### ISSUE TITLE
GeoPDF Export - step forward using Time Controller interval

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#4992]
When the user `Plays` a GeoPDF we control the time interval between each step forward (probably a second).

We should instead take this value from the TimeController `Step Interval` property.

Then `Play` can be faster or slower than one second.

**Comment by helenayele** [comment]
@IanMayo/ @saulhidalgoaular  , after I generate the track into GeoPDF what could I play the pdf file and test this issue?

**Comment by helenayele** [comment]
We are now taking the time interval from small step property rather than `Step Interval`, right ?
[![Image from Gyazo](https://i.gyazo.com/c99adfd07debb5b8f216aced146ef797.gif)](https://gyazo.com/c99adfd07debb5b8f216aced146ef797)
here is the step property:
[![Image from Gyazo](https://i.gyazo.com/79ae15fd8b1d99d59caaa67d29047550.png)](https://gyazo.com/79ae15fd8b1d99d59caaa67d29047550)

**Comment by IanMayo** [comment]
@helenayele = there are two intervals:
1. The pause in real-time between each step forwards for the animated replay (step interval in Debrief)
2. How far the time-label moves forward each step (Small step in Debrief)

**Comment by helenayele** [comment]
Okay clear thanks! 
One more thing I see an issue is, it exports invisible positions that are invisible  in debrief.
See the below screenshots:
In full debrief I filtered to some time period so that I want to see only those selected positions:
[![Image from Gyazo](https://i.gyazo.com/9ddc0c9f5b40956c10966efa37b2b15b.png)](https://gyazo.com/9ddc0c9f5b40956c10966efa37b2b15b)

And I exported it to GeoPDF file and looks like:
[![Image from Gyazo](https://i.gyazo.com/f5804b2cec8db99a4837d6f969d60710.gif)](https://gyazo.com/f5804b2cec8db99a4837d6f969d60710)
See that it shows all the positions of the track.
Is this by design?

**Comment by IanMayo** [comment]
No, it should only export visible positions.  That's one for @saulhidalgoaular .

**Comment by helenayele** [comment]
Okay @IanMayo  thanks.
@saulhidalgoaular , one more thing to correct is the end time shown on the pdf file.
if we see the above example,  small step = 5 min as example , and if the tracks end time = 121141, the track ends playing in GeoPDF file on 121140 not on 121141.
[![Image from Gyazo](https://i.gyazo.com/a0f1f13262a206cf8c4106fd3179b8be.png)](https://gyazo.com/a0f1f13262a206cf8c4106fd3179b8be)
